mods/cnc/sequences/aircraft.yaml

```diff
@@ -21,6 +21,9 @@ tran:
 	open:
 		Start: 32
 		Length: 4
+	close:
+		Start: 35
+		Length: -4
```

Comment:
From Discord: negative lengths only accidentally work in this case, and should not be used.

Reply:
Replaced it with individual frames.